File: JAVA_COURSE/KOTLIN/11_OvalApplication/src/edu/javacourse/ui/OvalFrame.java
package edu.javacourse.ui;

import javax.swing.JFrame;

public class OvalFrame extends JFrame
{
    public OvalFrame(String s) {
        // Создаем объект типа OvalComponent
        super(s);
        OvalComponent oc = new OvalComponent(50);
        // Испольщзуем метод класса JFrame для добавления
        // компонента на главную панель.
        add(oc);
        OvalComponent oc1 = new OvalComponent(5);
         add(oc1);
        // Устанавливаем координаты и размеры окна
        setBounds(200, 200, 300, 250);
    }
    
}

File: JAVA_COURSE/KOTLIN/11_OvalApplication/src/kotlincourse/ui/OvalFrame.kt
package kotlincourse.ui

import javax.swing.*

class OvalFrame(s: String) : JFrame(s) {
    init {
        val oc = OvalComponent(50)
        // Испольщзуем метод класса JFrame для добавления
        // компонента на главную панель.
        add(oc)
        val oc1 = OvalComponent(5)
        add(oc1)
        // Устанавливаем координаты и размеры окна
        setBounds(200, 200, 300, 250)
    }// Создаем объект типа OvalComponent

}
